mscore/exportdialog.cpp

```diff
@@ -80,8 +80,8 @@ ExportDialog::ExportDialog(Score* s, QWidget* parent)
       Q_ASSERT(fileTypeComboBoxModel != nullptr);
 # ifndef USE_LAME
       // Disable .mp3 option if unavailable
-      QStandardItem* mp3Item = fileTypeComboBoxModel->item(3);
-      mp3Item->setFlags(audioItem->flags() & ~Qt::ItemIsEnabled);
+      QStandardItem* audioItem = fileTypeComboBoxModel->item(3);
+      audioItem->setFlags(audioItem->flags() & ~Qt::ItemIsEnabled);
```

Comment:
Whxy this change? It does conflict if `USE_LAME` **and** `HAS_AUDIOFILE` is defined

Reply:
No they doesn't. They just local variables. In old code audioItem wasn't previously declared.

Reply:
Anyway, more important it to get it to build first ;-)

Reply:
I've fixed it myself

Reply:
Let me know whether you're happy with my changes